core/record/_record/RecordUpdateResult.js

```diff
@@ -0,0 +1,34 @@
+import * as RecordUpdater from './recordUpdater'
+
+export default class RecordUpdateResult {
+  constructor({ record = {}, nodes = {} } = { record: null, nodes: {} }) {
+    this._record = record
+    this._nodes = nodes
+  }
+
+  get record() {
+    return this._record
+  }
+
+  set record(record) {
+    this._record = record
+  }
+
+  get nodes() {
+    return this._nodes
+  }
+
+  getNodeByUuid(uuid) {
+    return this._nodes[uuid]
+  }
+
+  addNode(node) {
+    this._nodes[node.uuid] = node
+    this._record = RecordUpdater.assocNode(node)(this._record)
+  }
+
+  merge(updateResult) {
```

Comment:
updateResult is a function? o a result updated?

Reply:
same question for the name of the function

Reply:
how about a name self-explanatory instead of a comment in another place of the code?